hardhat.config.ts

```diff
@@ -107,7 +107,12 @@ const config: HardhatUserConfig = {
     },
     stabilitytestnet: {
       ...networkConfig,
-      url: "https://free.testnet.stabilityprotocol.com"
+      url: "https://free.testnet.stabilityprotocol.com",
+    },
+    stability: {
+      ...networkConfig,
+      // To get a API key, visit https://portal.stabilityprotocol.com
+      url: `https://gtn.stabilityprotocol.com/?api_key=${process.env.STABILITY_API_KEY ?? "tradeTrust"}`,
```

Comment:
Maybe dont expose the tradeTrust api key here, thanks

Reply:
Thanks for the advise! resolved